client/scss/components/_dropdown.scss

```diff
@@ -1,9 +1,10 @@
 // =============================================================================
 // Listing view smaller dropdowns
 // =============================================================================
+.c-dropdown.hidden {
```

Comment:
Adding `.hidden` to the `.c-dropdown` element does the job. do we need the extra specificity?

Reply:
Do you mean that we don't need this specific rule?

Reply:
Indeed. simply adding the `hidden` class to the html element already hides it

Reply:
> simply adding the hidden class to the html element already hides it

I've tried it but it doesn't seem to work.

Reply:
ok. must've been a fluke on my side. 🤔 let's keep it in. thank you for revisiting